values-ar/strings.xml

```diff
@@ -0,0 +1,142 @@
+        <!--HINTS-->
+            <string name="slide_arch_hint">التطبيق بشكل تلقائي يكتشف معمارية الجهاز</string>
+            <string name="slide_android_hint">التطبيق بشكل تلقائي يكتشف إصدار الأندرويد للجهاز</string>
+            <string name="slide_variant_hint">التطبيق بشكل تلقائي يكتشف النوع المثبت على الجهاز</string>
+        <!--VARIOUS-->
+            <string name="detected">أكتشف</string>
+
+    <!--______PREFERENCES______-->
+        <!--LABELS-->
+            <!--<string name="pref_header_install">Install (noun variant, not used in English)</string>-->
+            <!--<string name="pref_header_download">Download (noun variant, not used in English)</string>-->
+            <string name="new_folder">مجلد جديد</string>
+            <string name="pref_header_gapps_selection">إختيار-GApps</string>
+            <string name="action_settings">إعدادات</string>
+            <string name="label_wifi_only">Wi-Fi فقط</string>
+            <string name="label_download_dir">موقع التنزيل</string>
+            <string name="label_wipe_cache">مسح Cache/Dalvik</string>
+            <string name="label_download_md5">تنزيل MD5</string>
+            <string name="only_newest_version">تنظيف تلقائي للتنزيلات</string>
+            <string name="label_download_versionlog">تنزيل سجل الإصدار</string>
+            <string name="label_root_mode">إستخدام أذن الروت</string>
+        <!--SUMMARIES-->
+            <string name="summary_wifi_only">التنزيل عبر Wi-Fi فقط</string>
+            <string name="explanation_newest_version">حذف الإصدار القديمة إذا تم تنزيل نوع جديد متشابه</string>
+            <string name="explanation_download_md5">تحميل MD5-Checksum للتحقق من اكتمال الحزمة</string>
+            <string name="explanation_download_versionlog">تنزيل سجل الإصدار لقائمة محتوى الحزمة</string>
+            <string name="summary_root_mode">منح أذن الروت لتفعيل التثبيت والتعديل ومسح إعدادات cache/dalvik</string>
+            <string name="explanation_wipe_dalvik">فرض إعادة التحسين للتخزين المؤقت في الإقلاع التالي. يأخذ المزيد من الوقت، ولكن يساعد على منع المشاكل المحتملة.</string>
+        <!--FILECHOOSER (EXTERNAL LIBRARY)-->
+            <string name="confirm_label">تأكيد</string>
+            <string name="selected_folder_label">تحديد مجلد:</string>
+            <string name="create_folder_label">إنشاء مجلد</string>
+            <string name="create_folder_msg">إنشاء مجلد جديد بالاسم "%1$s"?</string>
```

Comment:
I am not sure if this went correct (I have no experience with right-to-left mixing with left-to-right).
The file-picker library at least expects a `%1$s` that it can replace with the foldername